opencensus/spanner/main.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample spanner contains a sample application that
+// uses Google Spanner Go client, and reports metrics
+// and traces for the outgoing requests.
+package main
+
+import (
+	"context"
```

Comment:
We're still using x/net/context across this repo. Can you change it?

Reply:
Done